app/models/content_block.rb

```diff
@@ -6,11 +6,33 @@ class ContentBlock < ApplicationRecord
 
   has_one_attached :file
   has_many :content_versions, dependent: :restrict_with_error
+  belongs_to :target_version, optional: true
```

Comment:
replace with ` belongs_to :target_version`

`target_version` is not optional for `content_block`